include/ibamr/MarangoniSurfaceTensionForceFunction.h

```diff
@@ -191,7 +191,7 @@ class MarangoniSurfaceTensionForceFunction : public SurfaceTensionForceFunction
     /*!
      * Marangoni coefficient.
      */
-    double d_marangoni_coefficient = std::numeric_limits<double>::signaling_NaN();
+    double d_marangoni_coefficient = 0.0;
```

Comment:
Why was this reverted? It's been long enough that I don't remember why we set this to NaN but I think it was done deliberately.

Reply:
Even I forgot why did we revert to zero value. But me and Amneet discussed it over Zoom and decided to keep it as zero.